package.json

```diff
@@ -12,7 +12,7 @@
         "electron": "3.0.2",
         "electron-edge-js": "8.3.8",
         "electron-localshortcut": "3.1.0",
-        "gpii-windows": "0.3.0-dev.20191017T162536Z.acb3d40",
+        "gpii-windows": "stegru/windows#GPII-4135",
```

Comment:
Cheers - can you please update this to the latest windows release of 0.3.0-dev.20191106T150515Z.cadcb7c which should incorporate the changes from @stegru

Reply:
Done 💯